### DISCUSSION TITLE
about time

### DISCUSSION THREAD

**Discussion by fishjoe** [micropython/micropython#10430] [RP2040 / Pico] [answered]
I'm in New Zealand. The local time is (2023, 1, 7, 9, 53, xx, xx, x)
I use ```ntptime.timeset()``` Then run ```time.localtime()``` it becomes (2023, 1, 6, 20, 53, xx, xx, x)
I machine.reset() then localtime is back to  (2023, 1, 7, 9, 53, xx, xx, x)

Question:
1. Is there a way set this back to local time without hard reset? ntptime.timeset(timezone=13) did not work.
2. Why reset brings me back to New Zealand localtime. How does the pico 2 know? If I bring the same board to say Australia, will it reset to  (2023, 1, 7, 7, 53, xx, xx, x) instead? 
3. 
I tried to use 
```rp2.counrty("AU")
time.localtime()
```
it still come back as NZ time. Nothing changed. By the way after  ```machine.reset()``` , ```rp2.country()``` result back to NZ again. Therefore, what is in the chipset made this chip into NZ chip? 

Many thanks.

**Comment by scruss** [comment, marked as answer]
Are you using Thonny? By default, it sets your attached MicroPython device to your localtime.

ntptime.settime() doesn't take parameters or know anything about timezone. You will always get UTC

**Reply by fishjoe** [reply]
thank you that's really helpful.
